src/platform/StdSchedulerWin32.cpp

```diff
@@ -87,9 +87,8 @@ bool StdScheduler::DoScheduleProcs(int iTimeout)
 	// Iterate over the index because procedures may be added or removed during execution
 	// (If they are removed, we skip one execution, which doesn't really matter in practice)
 	auto tNow = C4TimeMilliseconds::Now();
-	for (size_t i_proc = 0u; i_proc < procs.size(); ++i_proc)
```

Comment:
This transformation is invalid, as the comment above explains. There are a couple more replacements like this one throughout the patch where I don't know whether iterator invalidation is an issue. I guess we'll see if anything crashes.